tests/CMakeLists.txt

```diff
@@ -0,0 +1,31 @@
+project(Tests)
+
+include(${CMAKE_CURRENT_SOURCE_DIR}/../cmake/test_base.cmake)
```

Comment:
That's not necessary - this cmake file shouldn't really be standalone, so you can define this common utility in cmake one level higher.